index.js

```diff
@@ -939,10 +940,39 @@ module.exports = {
         formData.append('remote_path', path);
         formData.append('client_json', JSON.stringify(client_json))
         formData.append('encrypt', encrypt)
+
         const response = await utils.postReq(url, formData, option);
         return response
     },
 
+    syncUploadObject: async function(feed,remote_path,allocation,client_json,encrypt,delay = 5, option=null){
```

Comment:
We have not enforced prettier on js-client-sdk, but even if we don't do it, it'd be nice to attend to some basic patterns:

```suggestion
    syncUploadObject: async function (feed, remote_path, allocation, client_json, encrypt,delay = 5, option = null) {
```
If you can do the same for the rest of the changes, it'd be great!